### ISSUE TITLE
Theme desync on generated maps

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#547]
```
What steps will reproduce the problem?
1. In online game choose generated maps.
1. Click on map preview to generate a new map and theme.
2. Start the game.

What is the expected output? What do you see instead?
All players in the room should see the same theme but they don't. It causes 
desync.

What version of the product are you using? On what operating system?
hg tip 8579:90f445317e8a
Arch linux 64bit, opponent OS X

Please provide any additional information below.
Always reproducable.
```

Original issue reported on code.google.com by `ben...@unit22.org` on 1 Mar 2013 at 12:00

**Comment by GoogleCodeExporter** [comment]
```
Appears that other players stay on the old theme. No theme message emitted.

```

Original comment by `kyberneticist@gmail.com` on 1 Mar 2013 at 12:40

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 253750e192e7.
```

Original comment by `drew1313...@gmail.com` on 1 Mar 2013 at 8:51
* Changed state: **Fixed**
